crates/noirc_frontend/src/hir/resolution/import.rs

```diff
@@ -152,6 +152,7 @@ fn resolve_name_in_module(
             ModuleDefId::FunctionId(_) => panic!("functions cannot be in the type namespace"),
             // TODO: If impls are ever implemented, types can be used in a path
             ModuleDefId::TypeId(id) => id.0,
+            ModuleDefId::TypeAliasId(_) => panic!("type aliases cannot be in the type namespace"),
```

Comment:
They can't?

I think we can have cases where type aliases resolve to structs which can have methods. I'll need to double check if we already support this or not though. We may be able to get away with delaying this until a future PR.